reactor-core/src/test/java/reactor/core/scheduler/SchedulersMetricsTest.java

```diff
@@ -0,0 +1,127 @@
+package reactor.core.scheduler;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SchedulersMetricsTest {
+
+	final SimpleMeterRegistry simpleMeterRegistry = new SimpleMeterRegistry();
+
+	@Before
+	public void setUp() {
+		Metrics.addRegistry(simpleMeterRegistry);
+		Schedulers.enableMetrics();
+	}
+
+	@After
+	public void tearDown() {
+		Schedulers.disableMetrics();
+		Metrics.globalRegistry.forEachMeter(Metrics.globalRegistry::remove);
```

Comment:
with your latest change, this line is not needed anymore

Reply:
I'd prefer to keep it to ensure tests like the last one, which adds `Meter` separately, have no side effects on other tests.